.github/workflows/pr.yml

```diff
@@ -397,9 +398,32 @@ jobs:
           python-version: ${{matrix.python}}
         if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook'
 
+    #   - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Seem to be some more temp changes in the test files?

Reply:
Yes, all of the YML files can be ignored,